### DISCUSSION TITLE
Supabase - Library Marketplace

### DISCUSSION THREAD

**Discussion by mcamendoza1** [supabase/supabase#7455] [Feature Requests]
Will there be a chance to incorporate supabase and supabase/auth-helpers-nextjs to the sdk?

**Comment by silentworks** [comment]
I'm not sure what you are asking here as you need the supabase-js library in order to use the auth-helpers.

**Reply by mcamendoza1** [reply]
I mean, if there's a chance that the supabase team will include it/promote it in the supabase codebase.

Also would like to share my idea, what if there's a marketplace of the opensource library that was built on top of supabase so that developers can easily integrate their system to supabase. It can also help and support the community by getting paid or donations. 
Have seen this kind of model in directus and strapi. This is just an idea  :blush:
